### DISCUSSION TITLE
Backing up original firmware without STM32-CubeProgrammer

### DISCUSSION THREAD

**Discussion by ghost** [OpenI6X/opentx#282] [General]
Is there a way to create a backup on the EdgeTX Buddy site, or is the STM32 programmer the only option to create a backup?  Is there a site that has the original firmware to download if not?  I suppose I could always just download stm32 either way

**Comment by ajjjjjjjj** [comment]
EdgeTX Buddy can only write AFAIK.
Original firmware in the attachment:
[flysky_i6x_stock.zip](https://github.com/OpenI6X/opentx/files/9311451/flysky_i6x_stock.zip)
